config/locales/zh-CN.yml

```diff
@@ -0,0 +1,632 @@
+zh-CN:
+  articles:
+    read_on: '阅读'
+    selection: "文章"
+  formtastic:
+    yes: '是'
+    no: '否'
+    create: '创建 %{model}'
+    update: '更新 %{model}'
+    submit: '保存 %{model}'
+    cancel: '取消 %{model}'
+    reset: '重置 %{model}'
+    required: '必填'
+  errors:
+    messages:
+      already_confirmed: "已经确认，请尝试登录"
+      confirmation_period_expired: "需要在 %{period}之内确认, 请申请一个新的"
+      expired: "已过期, 请申请一个新的"
+      not_found: "未找到"
+      not_locked: "未锁定"
+      not_saved:
+        one: "发生了一个错误导致无法保存 %{resource} :"
+        other: "发生了%{count} 个错误导致无法保存 %{resource}: "
+  devise:
+    confirmations:
+      confirmed: "你的电子邮件地址已经被成功确认。"
+      send_instructions: "你将在几分钟内收到一封电子邮件，介绍如何确认你的电子邮件地址。"
+      send_paranoid_instructions: "如果你的电子邮件地址存在于我们的数据库中，你将在几分钟内收到一封电子邮件，其中包含如何确认你的电子邮件地址的说明。"
+    failure:
+      already_authenticated: "你已经登录。"
+      inactive: "你的账户尚未激活。"
+      invalid: "错误的 %{authentication_keys} 或密码。"
+      locked: "你的账户被锁定。 "
+      last_attempt: "在账户被锁定之前你还有最后一次机会尝试。"
+      not_found_in_database: "错误的 %{authentication_keys} 或密码。"
+      timeout: "你的会话已过期。 请重新登录以继续。"
+      unauthenticated: "你需要登录或注册才能继续。"
+      unconfirmed: "你必须要确认你的电子邮件地址才能继续。"
+    mailer:
+      confirmation_instructions:
+        subject: "确认指南"
+      reset_password_instructions:
+        subject: "重置密码指南"
+      unlock_instructions:
+        subject: "解锁指南"
+      password_change:
+        subject: "密码已更改"
+    omniauth_callbacks:
+      failure: "由于%{reason}，无法从%{kind} 认证。"
+      success: "成功从 %{kind} 认证帐户。"      
+    passwords:
+      no_token: "如果没有密码重置邮件无法访问这个页面。如果你是从密码重置邮件访问过来的，请确认输入的网址完整。"
+      send_instructions: "在几分钟内你会收到一封密码重置邮件。"
+      send_paranoid_instructions: "如果我们的数据库中保存有你的电子邮件地址，你会在几分钟内收到一封如何恢复密码的邮件。"
+      updated: "你的密码已被成功更改，你现在已经登录。"
+      updated_not_active: "你的密码已被成功更改。"      
+    registrations:
+      updated: "Your account has been updated successfully."
```

Comment:
Here is a translation missing.

Reply:
--> https://github.com/ikuseiGmbH/Goldencobra/commit/632e10be7448b238e28989423922d8b3467376aa